src/Atmos/Model/AtmosModel.jl

```diff
@@ -0,0 +1,122 @@
+
+import CLIMA.DGmethods: BalanceLaw, dimension, vars_aux, vars_state, num_state_for_gradtransform,  vars_gradtransform, vars_diffusive,
+  flux!, source!, wavespeed, boundarycondition!, gradtransform!, diffusive!,
+  init_aux!, init_state!, preodefun!
+
+struct AtmosModel{T,M,R} <: BalanceLaw
+  turbulence::T
+  moisture::M
+  radiation::R
```

Comment:
Other categories you'd want here eventually: 

- gravity_waves
- cloud_microphysics
- SGS_model (Smagorinsky-Lilly etc.; not sure what the best name is, as gravity_waves, turbulence/convection etc. are strictly also all SGS models)

'Turbulence' should also be something broader, e.g., 'turbulence-convection'. Or two categories, 'turbulence' and 'convection', for which we currently are using one scheme.